core/domain/summary_services.py

```diff
@@ -150,8 +179,15 @@ def get_learner_collection_dict_by_id(
 
 
 def get_displayable_collection_summary_dicts_matching_ids(collection_ids):
-    """Returns a list with all collection summary objects that can be
-    displayed on the library page as collection summary tiles.
+    """Returns a list of dict with all collection summary objects from
```

Comment:
Returns a list of collection summary dicts corresponding to the given collection ids.